### PR TITLE
[T] (quests) busy_bees

### PR DESCRIPTION
**Pull request by nai200**
Co-authored-by: 琳琳 <604896778@qq.com>
Co-authored-by: 晟鼎 <xiaodingyao@126.com>
Co-authored-by: 努力 <m18344860937@163.com>
Co-authored-by: invisibility <chenzf2020@outlook.com>
Co-authored-by: lee <527132183@qq.com>
Co-authored-by: one four-泷羽 <2148785875@qq.com>
Co-authored-by: SeeleVollerei <yunanwang2424@outlook.com>
Co-authored-by: xin_yi_hl <1012737146@qq.com>

### CODE REVIEW

File: config/ftbquests/quests/chapters/busy_bees.snbt [hunk 1]
```diff
@@ -504,15 +504,15 @@
...
-			subtitle: "Get Silky Comb"
+			subtitle: "获得丝滑蜂巢"
 			description: [
-				"Finally, I've eliminated the need for me to interact with the Spiders here."
+				"最后，我已经消除了在这里与蜘蛛互动的需要。"
 				""
-				"Instead, I can get my &6String&r from &bSilky Bees&r!"
-			]
+				"相反，我可以通过&b丝滑蜜蜂&r得到&6线&r！"
+
```
Comment (Ricky-fight) [inline config/ftbquests/quests/chapters/busy_bees.snbt:515]:
```suggestion
			]
```

File: config/ftbquests/quests/chapters/busy_bees.snbt [hunk 2]
```diff
@@ -1623,19 +1623,19 @@
...
-			title: "Vitamin Bee 1"
+			title: "维生素B1"
 			icon: "productivebees:gene_bottle"
 			x: -1.5d
 			y: -0.5d
-			subtitle: "Squash a Bee"
+			subtitle: "压碎蜜蜂"
 			description: [
-				"&6Every Bee is unique.&r Different bees have different attributes, which dictate how they bee-have."
+				"&6每只蜜蜂都是独一无二的。&r不同蜜蜂有不同属性，这决定了它们如何饲养。
```
Comment (Ricky-fight) [inline config/ftbquests/quests/chapters/busy_bees.snbt:1632]:
```suggestion
				"&6每只蜜蜂都是独一无二的。&r不同蜜蜂有不同属性，这决定了它们如何饲养。"
```

**Review by Ricky-fight — COMMENTED** [review]
建议在vsc上下载一个snbt文件格式的插件

**Comment by nai200** [comment]
检查发现协作出错
更改如下：
删除 one four-泷羽 [2148785875@qq.com](mailto:2148785875@qq.com)
增加 森绿小羽 2148785875@qq.com
增加 藏青 2291848664@qq.com

**Pull request merged**